package.json

```diff
@@ -108,6 +109,7 @@
     "react-palm": "^1.1.2",
     "react-redux": "^4.0.0",
     "react-tooltip": "^3.10.0",
+    "react-virtualized": "^9.21.0",
```

Comment:
where are we using react-virtualized in the source code?

Reply:
i think i accidentaly ran yarn add react-virtualized in the root folder